core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -4234,11 +4234,12 @@ private void createSymbolMapWriter(CharSequence name, long columnNameTxn, int sy
                 denseSymbolMapWriters.size(),
                 txWriter
         );
+        denseSymbolMapWriters.add(w);
```

Comment:
i wonder if we should just close the writer there and then and let exception fly? Having distressed writer in the data structures is no good

Reply:
@bluestreak01 wdyt now?